ui/js/graphs.js

```diff
@@ -299,12 +299,14 @@ perf.controller('GraphsCtrl', [
     function plotGraph() {
       // synchronize series visibility with flot, in case it's changed
       $scope.seriesList.forEach(function(series) {
-        series.flotSeries.points.show = series.visible;
+          series.flotSeries.points.show = series.visible;
+          series.active = (!series.subtestSignatures || $scope.myMeasure === 'mean');
+          series.blockColor = series.active ? series.color : "grey";
```

Comment:
Could you move these new lines two spaces to the left, to be consistent with the indentation in the rest of the file?